packages/form-layout/src/vaadin-form-layout.js

```diff
@@ -269,7 +266,7 @@ class FormLayout extends ElementMixin(ThemableMixin(mixinBehaviors([IronResizabl
     super.connectedCallback();
 
     beforeNextRender(this, this._selectResponsiveStep);
-    beforeNextRender(this, this.updateStyles);
+    beforeNextRender(this, () => this.__updateLayout());
```

Comment:
nit: I would prefer changing this to `requestAnimationFrame` later. Not a blocker.

Reply:
Done